debezium-core/src/main/java/io/debezium/relational/RelationalDatabaseConnectorConfig.java

```diff
@@ -454,6 +535,23 @@ public static DecimalHandlingMode parse(String value, String defaultValue) {
                     + " If you are excluding a lot of tables the default behavior should work well.")
             .withDefault(false);
 
+    public static final Field SNAPSHOT_ISOLATION_MODE = Field.create("snapshot.isolation.mode")
```

Comment:
let's introduce a new config as it will provide more flexibility to users 😊

Also, could you please suggest if I need to add this new field in the documentation anywhere? (apart from doc files present in this PR)

Reply:
@govi20 I don't think a differnt doc palces is needed.

Reply:
addressed this review comment.